### PR TITLE
Initial Yasson website at Eclipse.

### PR DESCRIPTION
**Pull request by Tomas-Kraus**
Signed-off-by: Tomas Kraus <Tomas.Kraus@oracle.com>

### CODE REVIEW

**Comment by aguibert** [comment]
hi @Tomas-Kraus, glad the website material is being migrated over! However, looks like a lot of changes are still needed before this can be merged. I assume the PR is just a placeholder for now and you will let us know when you want a review on this?

**Comment by Tomas-Kraus** [comment]
@aguibert Hi, please can you point to some specific issues you saw in this PR? I just wanted to prepare some simple template + latest javadoc from mvn central.

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +0,0 @@
-language: java
```
Comment (aguibert) [inline .travis.yml]: This file should not be deleted, we are still using TravisCI

File: docs/api/allclasses-frame.html [hunk 2]
```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```
Comment (aguibert) [inline docs/api/allclasses-frame.html]: These javadoc files are generated by the javadoc compiler, so they should not be checked in

File: pom.xml [hunk 3]
```diff
@@ -1,555 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aguibert) [inline pom.xml]: we still need a pom.xml obviously...

File: src/main/java/org/eclipse/yasson/FieldAccessStrategy.java [hunk 4]
```diff
@@ -1,28 +0,0 @@
-package org.eclipse.yasson;
```
Comment (aguibert) [inline src/main/java/org/eclipse/yasson/FieldAccessStrategy.java]: why all these files in src/main being removed?

**Pull request merged**